### PR TITLE
Add blog post for 10.9.0 testing

### PR DESCRIPTION
**Pull request by joshuaboniface**

### CODE REVIEW

**Comment by felix920506** [comment]
Testing instructions should really be in the docs. I think is fine to include them in a blog post but the need to test things won't just go away after 10.9 release.

**Comment by gnattu** [comment]
I don't think we do have platform-specific unstable documents on how to run unstable builds and the testing instructions are very docker and debian centric.

File: blog/2024/03-25-testing-10.9.0/index.mdx [hunk 1]
```diff
@@ -0,0 +1,71 @@
+---
+title: Testing 10.9.0
+description: We are now in feature freeze for 10.9.0; here's how to help test!
+authors: joshuaboniface
+date: 2024-03-25
```
Comment (nielsvanvelzen) [inline blog/2024/03-25-testing-10.9.0/index.mdx:5]: Don't forget to update this before merging

File: blog/2024/03-25-testing-10.9.0/index.mdx [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+
+Finally, install the unstable version and run it. The upgrade should happen seamlessly in the background, and you'll be able to log in to your Jellyfin instance normally after this point. Ensure you perform a *hard* refresh in your browser, and restart all clients.
+
+Once 10.9.0 is fully released, you can switch back easily by reinstalling the new stable version, and changing back to the stable plugin manifest (URL "https://repoo.jellyfin.org/files/plugin/manifest.json").
```
Comment (nielsvanvelzen) [inline blog/2024/03-25-testing-10.9.0/index.mdx:49]:
```suggestion
Once 10.9.0 is fully released, you can switch back easily by reinstalling the new stable version, and changing back to the stable plugin manifest (URL "https://repo.jellyfin.org/files/plugin/manifest.json").
```

repooooooo

**Review by thornbill — APPROVED** [review]

**Pull request merged**